packages/apollo/src/fetch-schema.ts

```diff
@@ -18,47 +20,74 @@ import { EndpointConfig } from "./config";
 import { engineLink, getIdFromKey } from "./engine";
 import { SCHEMA_QUERY } from "./operations/schema";
 
-const introspection = gql(introspectionQuery);
+const introspection = gql(getIntrospectionQuery());
 
-export async function fromFile(
+async function buildIntrospectionSchemaInLocalGraphQLContext(
+  source: string | Source
+): Promise<IntrospectionSchema> {
+  const schema: GraphQLSchema = buildSchema(source);
+  const executionResult: ExecutionResult = await graphql(
```

Comment:
I know this is unrelated to this PR, but I still find it confusing to import `execute` from `graphql-js` as `graphql`, while there is also a `graphql` export.

Reply:
This is actually no longer the case and this _really_ is `graphql` from `graphql`.  I did try to fix that myself by aliasing `execute` to `graphqlExecute` instead, but @jbaxleyiii 	took it a step further and just used `graphql`.

That said, what you're saying was still true when this logic still lived in [`schema/publish`](https://github.com/apollographql/apollo-cli/pull/614/files/c3054817ecde88f2ec4af6781c2368bb4c71e5e1#diff-1d42d3c5aff25031f2706605c8de0bb9L9) and [`schema/check`](https://github.com/apollographql/apollo-cli/pull/614/files/c3054817ecde88f2ec4af6781c2368bb4c71e5e1#diff-2364a0900ee1fb8e10783fc35eebddceL10).